configs/avax.json

```diff
@@ -2,26 +2,48 @@
   "index_name": "avax",
   "start_urls": [
     {
-      "url": "https://docs.avax.network",
+      "url": "https://docs.avax.network/",
       "tags": ["prod"]
     },
     {
       "url": "https://docs-beta.avax.network",
```

Comment:
should `/` be added at the end of this line? or does it matter?

Reply:
Oops I forgot this one indeed, it will work anyway but you can add it if you want!